### ISSUE TITLE
bug: Email copy icon on hover looks distorted 

### ISSUE THREAD

**Issue by nithindavid** [chatwoot/chatwoot#5597]
**Describe the bug**

1. Open any conversation in chatwoot dashboard. 
2. Add an email to the contact if it's not there.
3. Hover the email address copy icon.
4. The hovered state looks elongated.

https://user-images.githubusercontent.com/1277421/195199729-773c5694-ec82-4529-a768-6fcc7a999b79.mov

**Possible fixes**
-  Use `variant="clear"` instead of `variant="link"`
- Use `size="tiny"` 
- Remove `icon` class

**Expectation**
1. On hover the button should look square

**Comment by simonpankovski** [comment]
Hi, I would like to work on this issue. I checked out the problem in my browser and changing the icon's link class results in it appearing as a square similar to the other icons below it. However I noticed that doing this also adds more vertical spacing to the overall email row which might not be desirable, as shown below:

![image](https://user-images.githubusercontent.com/46830637/195204773-441d0dd6-0297-4983-b88c-ba52813d7e47.png)

Either way I am happy to help resolve the issue.
